apps/dg/components/graph/plots/number_toggle_model.js

```diff
@@ -186,8 +186,18 @@ return {
    */
   childrenOfParent: function( iIndex) {
     var tParents = this.get('parentCases' ),
-        tParent = (iIndex < tParents.length) ? tParents[ iIndex] : null;
-    return tParent ? tParent.get('children' ).flatten() : [];
+        tParent = (iIndex < tParents.length) ? tParents[ iIndex] : null,
+        tChildren = tParent ? tParent.get('children') : [];
+    for (var i = 0; i < tChildren.get('length'); ++i) {
```

Comment:
Is there a reason not to use tChildren.forEach here?

Reply:
`tChildren` is modified inside the loop to achieve the recursive behavior. The behavior of `forEach` is not clear under these circumstances. Your comment points out the need for a comment to that effect, which I will add momentarily.